lang/en/lang.php

```diff
@@ -117,6 +117,8 @@
         'posts_post_description' => 'Name of the blog post page file for the "Learn more" links. This property is used by the default component partial.',
         'posts_except_post' => 'Except post',
         'posts_except_post_description' => 'Enter ID/URL or variable with post ID/URL you want to except',
+        'posts_except_category' => 'Except category',
+        'posts_except_category_description' => 'Enter ID or variable with category ID you want to except',
```

Comment:
Please change `want to except` to `want to exclude` in this one and the `posts_except_post_description`